src/with_ladj.jl

```diff
@@ -84,9 +84,19 @@ _with_ladj_on_mapped(map_or_bc::Function, y_with_ladj::Tuple{Any,Real}) = y_with
 function _with_ladj_on_mapped(map_or_bc::Function, y_with_ladj)
     y = map_or_bc(_get_y, y_with_ladj)
     ladj = sum(map_or_bc(_get_ladj, y_with_ladj))
+    #ladj = sum(_get_ladj, y_with_ladj)
     (y, ladj)
 end
 
+function _with_ladj_on_mapped_pullback(thunked_ΔΩ)
+    ys, ladj = ChainRulesCore.unthunk(thunked_ΔΩ)
```

Comment:
You don't have to qualify it, it is loaded by `using ChainRulesCore`.
```suggestion
    ys, ladj = unthunk(thunked_ΔΩ)
```